src/notebook/notebook/widget.ts

```diff
@@ -716,11 +832,31 @@ class Notebook extends StaticNotebook {
   }
 
   /**
+   * Update the active cell when a cell is added or removed.
+   */
+  private _updateActiveCell(): void {
+    let count = this.childCount();
```

Comment:
If the active cell index is set to -1, is the activeCellChanged event emitted with a null cell?